### PR TITLE
[Jobs] Artifacts: link to item in Artifacts screen

### PR DESCRIPTION
**Pull request by IvanVoznyi**

### CODE REVIEW

File: src/components/DetailsArtifacts/DetailsArtifacts.js [hunk 1]
```diff
@@ -69,7 +70,13 @@ const DetailsArtifacts = ({ selectedItem }) => {
...
+  return (
+    <DetailsArtifactsView
+      content={content}
+      match={match}
+      showPreview={showPreview}
+    />
+  )
 }
 
 DetailsArtifacts.propTypes = {
```
Comment (mavdryk) [inline src/components/DetailsArtifacts/DetailsArtifacts.js:82]: add needed propTypes

File: src/components/Artifacts/Artifacts.js [hunk 2]
```diff
@@ -13,7 +13,13 @@ import artifactsData from './artifactsData'
 import './artifacts.scss'
 import { generateArtifactPreviewData } from '../../utils/generateArtifactPreviewData'
 
-const Artifacts = ({ artifactsStore, fetchArtifacts, history, match }) => {
+const Artifacts = ({
+  artifactsStore,
+  fetchArtifacts,
+  removeArtifacts,
+  history,
+  match
```
Comment (mavdryk) [inline src/components/Artifacts/Artifacts.js:21]: sort props

File: src/components/DetailsArtifacts/DetailsArtifactsView.js [hunk 3]
```diff
@@ -12,7 +13,7 @@ import { ReactComponent as Popout } from '../../images/popout.svg'
 
 import './detailsArtifacts.scss'
 
-const DetailsArtifactsView = ({ content, showPreview }) => (
+const DetailsArtifactsView = ({ content, showPreview, match }) => (
```
Comment (mavdryk) [inline src/components/DetailsArtifacts/DetailsArtifactsView.js:16]: sort props

File: src/components/Table/TableView.js [hunk 4]
```diff
@@ -28,9 +28,13 @@ const TableView = ({
   workflows
 }) => {
   const actionsMenu = [
-    { label: 'View YAML', icon: <Yaml />, onClick: toggleConvertToYaml }
+    {
+      label: 'View YAML',
+      icon: <Yaml />,
+      onClick: toggleConvertToYaml,
+      visible: true
```
Comment (mavdryk) [inline src/components/Table/TableView.js:35]: Why do you need to specify `visible` property?

File: src/components/DetailsArtifacts/detailsArtifacts.scss [hunk 5]
```diff
@@ -32,9 +32,11 @@
       padding: 12px 26px;
 
       &:first-child {
+        flex-direction: column;
+        align-items: end;
```
Comment (mariana-furyk) [inline src/components/DetailsArtifacts/detailsArtifacts.scss:36]: its' more safe to use flex-end (cross browser support), and in this case, I believe you should use flex-start, not end (please review it in the browser)

File: src/components/DetailsArtifacts/detailsArtifacts.scss [hunk 6]
```diff
@@ -45,6 +47,12 @@
         flex: 0.3;
         min-width: 75px;
       }
+
+      a {
```
Comment (mariana-furyk) [inline src/components/DetailsArtifacts/detailsArtifacts.scss:51]: please use classes not tags to define some styles

**Pull request merged**